### PR TITLE
Added the youtube search and autoscroll feature 

### PR DESCRIPTION
**Pull request by muskankhedia**
#10 The youtube search has been added in this PR. Now the user can make youtube search using Personal Assistant, the syntax of search is **youtube <message>**.

#27 And also the auto scroll to the end of the page is been added.

The features are better explained in the video. To watch the video [click here.](https://drive.google.com/file/d/1HsSqY0d1eQBcXNGdpkaEek1D4IWD2BXy/view)

### CODE REVIEW

File: service/controllers/messages.go [hunk 1]
```diff
@@ -399,3 +421,60 @@ func processBingResponses(result string) []messageQueryBody {
...
+	lensubsl3 := len(subsl3)
+
+	var queryResult messageQueryBody
+	var queryResultArray []messageQueryBody
+	var mid int
+
+	for i := 0; i < len(result) - len(subsl); i++ {
+		mess := ""
+		if result[i : i + len(subsl)] == subsl {
+			fmt.Println("reached1")
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: if the code is working as expected, could you please remove these print statements

File: service/controllers/messages.go [hunk 2]
```diff
@@ -399,3 +421,60 @@ func processBingResponses(result string) []messageQueryBody {
...
+			length := i + len(subsl)
+			var last int
+			for j:=1; ; j++ {
+				if result[length + j: length + j + len(subsl2)] == subsl2 {
+					fmt.Println("reached2")
+					mid = length + j + len(subsl2)
+					fmt.Println(mid)
+					for k := 1; ; k++ {
+						if result[mid + k: mid + k + 2] == "\">" {
+							fmt.Println("reached3")
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: same

File: service/controllers/messages.go [hunk 3]
```diff
@@ -399,3 +421,60 @@ func processBingResponses(result string) []messageQueryBody {
...
+					}
+				if found {
+					queryResultArray = append(queryResultArray, queryResult)
+					break
+				}
+			}
+		}
+	}
+	return queryResultArray
+}
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: please give a new line at the end of the file

File: view/css/main.css [hunk 4]
```diff
@@ -39,14 +39,14 @@ header {
...
-    height: 75vh;
+    height: 65vh;
     overflow-y: scroll;
 }
 
 #stackArea {
     padding: 0;
     margin: 0;
-    max-height: 200%;
+    max-height: 2000%;
```
Comment (Harkishen-Singh) [inline view/css/main.css]: please justify this

File: service/controllers/messages.go (hunk 1 above)
Comment (muskankhedia) [reply service/controllers/messages.go]: Yaa sure.... sorry for the mistake

File: service/controllers/messages.go (hunk 2 above)
Comment (muskankhedia) [reply service/controllers/messages.go]: yup

File: service/controllers/messages.go (hunk 3 above)
Comment (muskankhedia) [reply service/controllers/messages.go]: yaa added

File: view/css/main.css (hunk 4 above)
Comment (muskankhedia) [reply view/css/main.css]: for the messages not to overlap.

**Pull request merged**